src/parser.js

```diff
@@ -53,6 +48,33 @@ function isJsonContentType(contentType) {
   }
 }
 
+function isMultiPartFormData(contentType) {
+  try {
+    const type = typer.parse(contentType);
+    return type.type === 'multipart' || type.subtype === 'form-data';
```

Comment:
Shouldn't this be `&&`? Similarly below.

Reply:
Yes, I've fixed this. The problem exists in another method too which I fixed, I copied the similar code and adapted for multipart.